src/main/config/simplified_tuning.c

```diff
@@ -34,59 +34,25 @@ static void calculateNewPidValues(pidProfile_t *pidProfile)
 {
     const pidf_t pidDefaults[FLIGHT_DYNAMICS_INDEX_COUNT] = {
             [PID_ROLL] = PID_ROLL_DEFAULT,
-            [PID_PITCH] = PID_ROLL_DEFAULT, // using the same defaults as roll
+            [PID_PITCH] = PID_PITCH_DEFAULT, // using the same defaults as roll
```

Comment:
Switching from Roll to Pitch Default here. How about changing the comment too. For reference and understanding what is the difference in this change?

Reply:
Ops, forgot to remove the comment, in the old code calculated roll and pitch pids were the same(based on the same roll defaults)

Reply:
Also this seems to include the commit / changes from 10508 but I guess it's to test this PR until squashed / rebased?

Any idea about the disabled filter sliders (cannot reset / enable) in configurator?

Reply:
Yes, I've done this upon the other branch